src/main/java/cs2103t/ip/duke/Duke.java

```diff
@@ -0,0 +1,367 @@
+package cs2103t.ip.duke;
+
+import java.io.*;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.time.LocalDate;
+
+import javafx.animation.PauseTransition;
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+import javafx.scene.layout.Region;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.text.Text;
+import javafx.util.Duration;
+
+public class Duke extends Application {
+
+    public static final String FOLDER_PATH = "../data";
+    public static final String FILE_PATH = "../data/duke.txt";
```

Comment:
Would it be better to make these two strings start with the PATH prefix? This would allow one to easily identify that these are PATH variables since they do belong in the same group.